tox.ini

```diff
@@ -149,5 +149,4 @@ commands =
 deps =
     -r resources/requirements_mypy.txt
 commands =
-    mypy napari/settings napari/types.py napari/plugins  --check-untyped-defs
-package = editable-legacy
```

Comment:
Did you check that it is obsolete now?

Reply:
tox always install the package until you not explicitly ask to not do this. https://tox.wiki/en/latest/config.html#skip_install